eZ/Bundle/EzPublishCoreBundle/Resources/config/url_checker.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+    ezpublish.url_checker.class: eZ\Bundle\EzPublishCoreBundle\URLChecker\URLChecker
+    ezpublish.url_checker.handler_registry.class: eZ\Bundle\EzPublishCoreBundle\URLChecker\URLHandlerRegistry
+    ezpublish.url_checker.handler.http.class: eZ\Bundle\EzPublishCoreBundle\URLChecker\Handler\HTTPHandler
+    ezpublish.url_checker.handler.mailto.class: eZ\Bundle\EzPublishCoreBundle\URLChecker\Handler\MailToHandler
```

Comment:
Symfony decided some time ago to drop `.class` parameters and there's no reason we shouldn't do it as well in new files.

Reply:
Done 44733c2824332f18ca89a46583160bece5118e21